common/buildings/hfe_buildings.txt

```diff
@@ -410,4 +526,1210 @@ hfe_building_thoth_module_industry = {
+			}
+		}
+		modifier = {
+			job_hfe_job_godforge_primitive_add = 1
+			job_primitive_laborer_add = 5
+			job_primitive_priest_2_add = 1
+			job_researcher_add = -4 # Negating the Relic stuff
+			job_soldier_add = -1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+			owner = {
+				OR = {
+					has_civic = hfe_civic_high_mechanicus
+					is_spiritualist = yes
+				}
+				NOT = { has_civic = civic_barbaric_despoilers }
+			}
+		}
+		modifier = {
+			job_noble_add = 1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+		}
+		modifier = {
+			planet_metallurgists_minerals_upkeep_mult = -0.5
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { is_ai = yes }
+		}
+		planet_districts_upkeep_mult = -0.75
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = {
+				NOT = {
+					is_country_type = primitive
+					is_primitive = yes
+					has_ethic = ethic_gestalt_consciousness
+				}
+			}
+#			OR = {
+#				owner = { has_country_flag = hfe_varelse_primitive_0 }
+#				owner = { has_country_flag = high_mechanicus_retained }
+#				owner = { has_country_flag = hfe_varelse_fedora_governance }
+#				owner = { has_government = hfe_gov_high_mechanicus }
+#			}
+		}
+		modifier = {
+			job_hfe_job_godforge_overlord_add = 1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			OR = {
+				NOT = { owner = { has_country_flag = hfe_varelse_primitive_0 } }
+				NOT = { owner = { is_country_type = primitive } }
+			}
+			NOT = { owner = { has_ethic = ethic_gestalt_consciousness } }
+		}
+		job_foundry_per_pop = 0.1
+		job_miner_per_pop = 0.04
+		job_odd_factory_worker_add = 4
+		job_odd_factory_worker_per_pop = 0.05
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_ethic = ethic_gestalt_consciousness }
+		}
+		modifier = {
+			job_alloy_drone_per_pop = 0.1
+			job_artisan_drone_add = 2
+			job_mining_drone_per_pop = 0.04
+			job_odd_factory_drone_add = 4
+			job_odd_factory_drone_per_pop = 0.05
+		}
+	}
+
+	resources = {
+		category = hfe_godforge
+		cost = {
+			minerals = 10000
+			alloys = 3000
+		}
+		produces = {
+			trigger = {
+				owner = {
+					OR = {
+						has_country_flag = hfe_varelse_primitive_0
+						has_country_flag = high_mechanicus_retained
+						has_country_flag = hfe_varelse_fedora_governance
+					}
+				}
+			}
+			alloys = 30
+			consumer_goods = 50
+			food = 30
+			engineering_research = 10
+		}
+		produces = {
+			trigger = {
+				owner = { has_country_flag = hfe_varelse_primitive_0 }
+				owner = { is_country_type = primitive }
+			}
+			consumer_goods = 20
+			food = 50
+		}
+		produces = {
+			trigger = {
+				NOT = {
+					owner = { has_country_flag = hfe_varelse_primitive_0 }
+				}
+			}
+			alloys = 20 # The rest is by the Jobs
+			consumer_goods = 10
+			engineering_research = 15
+		}
+		upkeep = {
+			trigger = {
```

Comment:
i do not see why it should have different upkeeps

Reply:
It depends on the nation and the nation's economy. This is the work of two months of precision, balance, and effort. I know what I am doing.